app/controllers/photos_controller.rb

```diff
@@ -1,15 +1,17 @@
 class PhotosController < ApplicationController
   before_action :set_photo, only: %i[ show edit update destroy ]
-  before_action :ensure_current_user_is_owner, only: [:destroy, :update, :edit]
+  before_action { authorize @photo || Photo }
+
 
   # GET /photos or /photos.json
   def index
     @photos = Photo.all
   end
 
   # GET /photos/1 or /photos/1.json
-  def show
-  end
+  #def show
+    #@photo
```

Comment:
Oh sorry I meant `@photo` is not needed. You can leave `def show end`, which you could say is not needed either, but not a bad thing to see.

Reply:
good job with your coding, looks clean. i don't think the edit and delete buttons are there yet, but I'm sure you're working on it. cheers! :)